### ISSUE TITLE
Warning message incomprehensible for end-user

### ISSUE THREAD

**Issue by thorstenkampe** [jazzband/docopt-ng#5]
Wenn I run my script without a necessary subcommand, `docopt-ng` issues a warning - shouldn't it be an error? - which in my opinion is not suitable for the enduser (`found unmatched (duplicate?) arguments [Argument(None, '...')]`.

Even I as the script writer, don't know what that exactly means. It looks to me like `docopt-ng` exposes internals from the parsing process which should not be given to the script user.
```
> python interpreter.py '.\10. PowerShell.ps1'
Warning: found unmatched (duplicate?) arguments [Argument(None, '.\\10. PowerShell.ps1')]
Usage:
 interpreter.py run <script> [-- <script_option> ...]
 interpreter.py doc [-s <text>] <script>
```

**Comment by loelkes** [comment]
I had the same issue but fixed it through trial and error. I had something like this:

```
Usage:
  script.py (--save | --load) FILE

Arguments:
  FILE                     

Options:
  --save FILE
  --load FILE
```
The error message was:
```
$ python script.py --save file.ini
Warning: found unmatched (duplicate?) arguments [Option(None, '--save', 1, 'file.ini')]
```
Removing ```FILE``` after ```--save``` in ```Options:``` did the trick for me.

**Comment by jambonrose** [comment]
The error message presented is confusing.

Provided the following:

```python
"""CLI

Usage:
  cli <arg1> <arg2>

"""
from docopt import docopt


def main():
    """MVE of confusing error"""
    arguments = docopt(__doc__, version="1.0.0")
    print(arguments)


if __name__ == "__main__":
    main()
```

The command line behavior is:

```console
$ cli 
Usage:
  cli <arg1> <arg2>

$ cli 1 2
{'<arg1>': '1',
 '<arg2>': '2'}

$ cli 1 
Warning: found unmatched (duplicate?) arguments [Argument(None, '1')]
Usage:
  cli <arg1> <arg2>
```

I note for clarity that the help and version features work despite being undocumented.

```console
$ cli --help
CLI

Usage:
  cli <arg1> <arg2>

$ cli -h
CLI

Usage:
  cli <arg1> <arg2>

$ cli --version
1.0.0
```

**Comment by traviswaelbro** [comment]
I was running into this issue as well and it ended up being because I was calling `docopt(..., options_first=True, ...)` and it was evaluating differently than I expected, which was causing the unclear error message. Removing that option resolved it in my case.

**Comment by FelixSchwarz** [comment]
This is really the most annoying issue in docopt-ng for me. The warning is also shown when a user mistypes an parameter (e.g. `--dryrun` instead of `--dry-run`) and the error message is almost impossible to understand as an end user.

**Comment by NickCrews** [comment]
I agree this is an issue, I would like to see it fixed. I am unlikely to implement it myself in any immediate time frame, but I will happily review any PRs people submit! Please start them small and get feedback quickly, I don't want you to waste your time. Thanks!

**Comment by NickCrews** [comment]
If you are affected by this and have opinions, you could take a look at https://github.com/jazzband/docopt-ng/pull/77. In particular, read the tests and verify that they are demonstrating the behavior that you would actually like.
